TransferMethod/Sources/View/Widget/AbstractWidget.swift

```diff
@@ -127,9 +128,27 @@ class AbstractWidget: UIStackView, UITextFieldDelegate {
 
     func textFieldDidBeginEditing(_ textField: UITextField) {
         textField.placeholder = nil
+        if let isValueMasked = field.fieldValueMasked, isValueMasked && !isValueUpdated {
+            textField.selectedTextRange = textField.textRange(from: textField.beginningOfDocument,
+                                                              to: textField.endOfDocument)
+        }
+    }
+
+    func textField(_ textField: UITextField,
+                   shouldChangeCharactersIn range: NSRange,
+                   replacementString string: String) -> Bool {
+        if let isValueMasked = field.fieldValueMasked, isValueMasked && !isValueUpdated {
+            textField.text = ""
+            isValueUpdated = true
```

Comment:
This should be outside of the if `isValueMasked = field.fieldValueMasked, isValueMasked ` block. isValueUpdated should be set to true for any other field as well that is being updated but should be set only once